exchange/src/exchange/tickers/manager/ticker_manager.cpp

```diff
@@ -41,6 +41,16 @@ log_match(const matching::stored_match& order)
         .Increment(order.quantity);
 }
 
+size_t
+EngineManager::match_order(const matching::stored_order& order)
+{
+    auto& ticker = get_engine(order.ticker);
+    std::vector<matching::stored_match> matches =
+        ticker.engine.match_order(ticker.orderbook, order);
+    std::ranges::move(matches, std::back_inserter(accum_matches_));
+    return matches.size();
```

Comment:
```suggestion
    auto matches = ticker.engine.match_order(ticker.orderbook, order);
    size_t num_matches = matches.size();
    
    std::ranges::move(matches, std::back_inserter(accum_matches_));
    return num_matches;
```

can't access something after you move it.

Reply:
good catch